src/python/WMCore/WorkQueue/DataStructs/WorkQueueElement.py

```diff
@@ -213,7 +213,7 @@ def statusMetrics(self):
     def passesSiteRestriction(self, site):
         """Takes account of white & black list, and input data to work out
         if site can run the work"""
-        # Don't check anything if useSiteListAsLocation/trusSiteLists is enabled
+        # Don't check anything if trustSitelists is enabled
         if self['NoLocationUpdate']:
```

Comment:
How is this NoLocationUpdate set ? It defaults to False, while standard TrustSiteLists modus would need it at True, ie. we really don't care about file locations at all.